pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/DeclarativePipeline2Test.java

```diff
@@ -0,0 +1,63 @@
+package org.jenkinsci.plugins.pipeline.modeldefinition;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.RealJenkinsRule;
+
+public class DeclarativePipeline2Test {
+    @Rule
+    public RealJenkinsRule r = new RealJenkinsRule();
```

Comment:
Any particular reason to create a `RealJenkinsRule`-based test? It doesn't hurt, but it is slower than normal `JenkinsRule` and I am not sure that we care much about the enhanced fidelity in this case.

Reply:
No particular reason. I wasn't aware of the `JenkinsRule`. I am going to use that instead 🙏

Reply:
I'll remove the test. Now it looks unnecessary, as other tests should cover if all is good with provided option